backend/tracim_backend/config.py

```diff
@@ -533,7 +533,7 @@ def _load_ldap_config(self) -> None:
         Load config for ldap related stuff
         """
         self.LDAP_URL = self.get_raw_config("ldap_url", "ldap://localhost:389")
-        self.LDAP_BASE_URL = self.get_raw_config("ldap_base_url")
+        self.LDAP_BASE_DN = self.get_raw_config("ldap_base_dn")
```

Comment:
`ldap_base_url` parameter is an error, this parameter is not used at all and visible only here (not used at all in code). Correct parameter named used in sample config is `ldap_base_dn`, this parameter come from tracimv1 legacy but is not more used (that's the reason why this error doesn't break anything).

We should decide how we handle this: 
- Is there some reason to retain `base_dn` ? I don't think so. We should think about removing it.

Reply:
It smells 👃 possible bug for ldap users. We should investigate what is necessary for ldap users to retain what it should be.

Reply:
> It smells nose possible bug for ldap users. We should investigate what is necessary for ldap users to retain what it should be.

As far i checked, automated test for ldap doesn't failed at all (they use a true provided ldap) and config param here is not found at all in source code, so i'm confident about ldap support (technically) but having a "not used" parameter is confusing.